package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "armstrong.hatband",
-    "version": "1.4.0",
+    "version": "1.4.0tt1",
```

Comment:
Although this pattern for patching has been done before, that was prior to how Python packaging community changed how they defined versions. This version string may upset newer versions of pip.

I just skimmed this, but the answer is probably in https://www.python.org/dev/peps/pep-0440/#public-version-identifiers